<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TQPayConfirmDlg</class>
 <widget class="QDialog" name="TQPayConfirmDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TQPayConfirmDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#wgt_main{
background:#262626;
border:1px solid #585858;
}
QLabel,QPushButton,QLineEdit{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
background:transparent;
color:#999999;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="wgt_main" native="true">
     <property name="styleSheet">
      <string notr="true">#wgt_title{
background:#2F2F2F;
}
#lbl_pay_passwd{
padding-left:39px;
}
#lineEdit_pay_passwd{
border:1px solid #585858;
border-radius:4px;
padding-left:10px;
}
#btn_confirm{
background-image: url(:/TQLive/images/btn_confirm.png);
background-repeat:no-repeat;
border:none;
}
#btn_confirm:hover{
background-image: url(:/TQLive/images/btn_confirm_hover.png);
}
#btn_confirm:pressed{
margin:1px;
padding:1px;
}
#btn_cancel{
background-image: url(:/TQLive/images/btn_cancel.png);
background-repeat:no-repeat;
border:none;
}
#btn_cancel:hover{
background-image: url(:/TQLive/images/btn_cancel_hover.png);
}
#btn_cancel:pressed{
margin:1px;
padding:1px;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>16</number>
      </property>
      <item>
       <widget class="QWidget" name="wgt_title" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lbl_title{
font-size:14px;
color:#4AB134;
}
#btn_close{
background-image: url(:/TQLive/images/pb_close_pressed.png);
background-repeat:no-repeat;
border:none;
}
#btn_close:hover{
background-image: url(:/TQLive/images/pb_close_normal.png);
}
#btn_close:pressed{
margin:1px;
padding:1px;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>19</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>9</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="lbl_title">
           <property name="text">
            <string>输入交易密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_close">
           <property name="minimumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>12</width>
             <height>13</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="wgt_header" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#lbl_payable_value,#lbl_balance_value{
color:#4AB134;
}
#btn_recharge{
background:#3F3F3F;
border-radius:2px;
color:#4AB134;
padding-left:0px;
padding-right:0px;
}
#btn_recharge:hover,#btn_recharge:pressed{
margin:1px;
padding:1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>40</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>29</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_payable">
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>需支付：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <item alignment="Qt::AlignHCenter">
              <widget class="QLabel" name="lbl_payable_value">
               <property name="text">
                <string>100</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_payable_unit">
               <property name="text">
                <string>探球币</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>29</number>
           </property>
           <item>
            <widget class="QLabel" name="lbl_balance">
             <property name="minimumSize">
              <size>
               <width>48</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>48</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>余额：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lbl_balance_value">
               <property name="text">
                <string>5</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lbl_balance_unit">
               <property name="text">
                <string>探球币</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="btn_recharge">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>去充值</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QLabel" name="lbl_pay_passwd">
          <property name="text">
           <string>交易密码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_pay_passwd">
          <property name="minimumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>210</width>
            <height>30</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>请输入交易密码</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>34</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_confirm">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cancel">
          <property name="minimumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>82</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TQLive.qrc"/>
 </resources>
 <connections/>
</ui>
